doc/releases/release-notes-4.1.rst

```diff
@@ -1,5 +1,28 @@
 :orphan:
 
+..
+  What goes here: removed/deprecated apis, new boards, new drivers, notable
+  features. If you feel like something new can be useful to a user, put it
+  under "Other Enhancements" in the first paragraph, if you feel like something
+  is worth mentioning in the project media (release blog post, release
+  livestream) put it under "Major enhancement".
+
+  No list of bugfixes, minor changes, those are already in the git log, this is
+  not a changelog.
+
+  Does the entry have a link that contains the details? Just add the link, if
+  you think it needs more details, put them in the content that shows up on the
+  link.
+
+  Are you thinking about generating this? Don't put anything at all.
+
+  Does the thing require the user to change their application? Put it on the
+  migration guide instead. (TODO: move the removed APIs section in the
+  migration guide)
+
+  Could the detailed paragraph that you are about to write go in the actual
+  project documentation instead and just be linked here? Do that instead.
```

Comment:
this deserves being moved higher up IMO -- people sometimes don't realize that they end up writing really good "documentation" that just happens to get a lot less long-term visibility than it should

Reply:
done